src/components/ClientCard/ClientCard.jsx

```diff
@@ -22,17 +26,20 @@ const ClientCard = props => (
 
 ClientCard.propTypes = {
   chat: PropTypes.object.isRequired,
+  config: PropTypes.object.isRequired,
   children: PropTypes.node.isRequired,
   setActiveChat: PropTypes.func.isRequired,
+  loadMessages: PropTypes.func.isRequired,
 };
 
 
-const mapStateToProps = (/* state */) => ({
-
+const mapStateToProps = state => ({
+  config: state.chat.config,
 });
 
 const mapDispatchToProps = dispatch => ({
   setActiveChat: chat => dispatch(setActiveChat(chat)),
+  loadMessages: (config, id) => loadMessages(dispatch, config, id),
```

Comment:
Honestly, was trying to find the better solution, I am not really a fan of the Application.jsx version, it feels weird to me? But its just a feeling. This way doesnt sit well with me either, but for whatever reason seemed less weird at the time... Again both are just feelings and not really justifiable, passing dispatch into an action I think is the root of the oddness, I've never seen it before, is it a common pattern?